src/javascript/extendable_audio.js

```diff
@@ -0,0 +1,46 @@
+/// <reference path="./extendable_audio.d.ts" />
+
+const SKIP_PROPERTIES = ['dispatchEvent', 'addEventListener', 'removeEventListener'];
+
+export class ExtendableAudio extends DocumentFragment {
+  constructor(url) {
+    this._audio = new Audio();
+    this._documentFragment = document.createDocumentFragment();
+    this._toSetUrl = url;
```

Comment:
Because we later override the `set url()` method, we want to give our subclass a chance to set up before we actually set it. This lets it create its fetchers etc.